### ISSUE TITLE
field with attribute widget list blocks content creation

### ISSUE THREAD

**Issue by loicginoux** [sveltia/sveltia-cms#126]
Given this config:
```yaml
collections:
    - name: "posts"
      i18n: true
      label: "Blog Posts"
      folder: "src/content/blog"
      create: true
      delete: true
      fields:
          - label: "Tags"
            name: "tags"
            widget: "list"
```
I should be able to create a new post with a tag list, but I have an error when saving the new post that blocks me from creating content.

<img width="1206" alt="Screenshot 2024-03-25 at 23 17 36" src="https://github.com/sveltia/sveltia-cms/assets/824936/06c8a348-3883-4e34-9bd2-b39fff5e90f9">

It does not happen when duplicating an existing post, and it is definitely an issue with this field because it works when I have other type of fields (string, boolean, text, object, etc..)

**Comment by kyoshino** [comment]
Thanks for reporting, @loicginoux! Will take a look now.

**Comment by kyoshino** [comment]
I guess you have i18n enabled. Can I see the i18n configuration?

**Comment by kyoshino** [comment]
Well, maybe this is your current config:
https://github.com/loicginoux/SDprices-blog2/blob/main/public/admin/config.yml

I don’t get any error with or without the `tags` field. 🤔

**Comment by kyoshino** [comment]
Okay, I think this is the same issue as #91. It’s fixed in v0.9.7 but [you’re using v0.9.0](https://github.com/loicginoux/SDprices-blog2/blob/main/public/admin/index.html#L54). Remove the version to use the latest version, then your problem should be solved!

```html
<script src="https://unpkg.com/@sveltia/cms/dist/sveltia-cms.js" type="module"></script>
```

**Comment by loicginoux** [comment]
Yes this is it, thank you and sorry for duplicate issue, I should have search more on closed issues. Nice work btw, keep it up !
